src/platform/Darwin/DnssdImpl.h

```diff
@@ -27,9 +27,15 @@
 #include <string>
 #include <vector>
 
+constexpr char kLocalDot[] = "local.";
+
+constexpr char kOpenThreadDot[] = "openthread.thread.home.arpa";
```

Comment:
How come there is no trailing`.` on this one?

Reply:
I checked with Ted. he said trailing . is more correct? he doesn't care either way. I told him we will go with the "arpa."

Reply:
Modified this to - "default.service.arpa." after discussion with Ted. he mentioned that he didn't think the domain name through :)